src/algorithms/simple2.jl

```diff
@@ -0,0 +1,127 @@
+struct SimpleBeeswarm2 <: BeeswarmAlgorithm
```

Comment:
Should we rename this?  Maybe make this `SimpleBeeswarm` and rename the current default?

Reply:
Yeah I think that would be good. Do you actually still want to keep the current default given that it causes overlaps?